migrations/versions/48c21f3e17d_initial_migration.py

```diff
@@ -0,0 +1,37 @@
+"""Initial migration
+
+Revision ID: 48c21f3e17d
+Revises: None
+Create Date: 2015-04-09 13:08:03.354601
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '48c21f3e17d'
+down_revision = None
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('title_register_data',
+    sa.Column('title_number', sa.String(length=10), nullable=False),
+    sa.Column('register_data', postgresql.JSON(), nullable=True),
+    sa.Column('geometry_data', postgresql.JSON(), nullable=True),
+    sa.PrimaryKeyConstraint('title_number')
+    )
+    op.create_table('title_numbers_uprns',
```

Comment:
any chance we can forget about the title_numbers_uprns in here? feeder doesn't write to it and api shouldn't be reading from it anymore